src/Tribe/Attendee_Registration/Shortcode.php

```diff
@@ -56,7 +56,10 @@ public function maybe_enqueue_scripts() {
 	public function render() {
 		ob_start();
 
-		echo tribe( 'tickets.attendee_registration.view' )->display_attendee_registration_shortcode();
+		/** @var Tribe__Tickets__Attendee_Registration__View $view */
+		$view = tribe( 'tickets.attendee_registration.view' );
+
+		echo $view->display_attendee_registration_shortcode();
```

Comment:
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$view'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$view'.

<sub>hash: 456613cfdef6f8dcdbbe32b1c9e56aa3</sub>